gammapy/maps/wcsnd.py

```diff
@@ -599,3 +600,50 @@ def smooth(self, radius, kernel='gauss', **kwargs):
         image = copy.copy(self)
         image.data = data
         return image
+
+    def make_cutout(self, position, radius, margin=None, mode="strict"):
+        """
+        Create a cutout of a WcsNDMap around a given direction.
+
+        Parameters
+        ----------
+
+        position : `~astropy.coordinates.SkyCoord`
+            Center position of the cutout box
+        radius : tuple of `~astropy.coordinates.Angle`
+            Angular sizes of the box
+        margin : `~astropy.coordinates.Angle`, optional
+            Additional safety margin. If specified must be same length as radius
+        mode : {'trim', 'partial', 'strict'}
+            Mode option for Cutout2D,  see http://docs.astropy.org/en/stable/api/astropy.nddata.utils.Cutout2D.html 
+
+        Returns
+        -------
+        cutout : `~gammapy.maps.WcsNDMap`
+            The cutout map itself
+        cutout_slices : tuple
+            Tuple of 1-dim slice objects
+
+        """
+
+        from astropy.nddata import Cutout2D
+        
+        size=radius+margin if margin else radius
```

Comment:
Write PEP8 clean code, i.e. have a space on the left and right of `=`.
Use an editor (like PyCharm or VS Code with Python extension) that does that automatically or via "format code" for you; using command line tools like `autopep8` is too painful.